pydynamo/test/sstable_test.py

```diff
@@ -0,0 +1,141 @@
+import unittest
+from ..storage.disk.sstable import SSTable
+from ..storage.error import StorageException
+from ..storage.disk.sstableiterator import SStableIterator
+from ..storage.disk.diskstore import DiskStore
+import tempfile
+
+
+class SStableTest(unittest.TestCase):
```

Comment:
you need add more tests on iterator for
1) empty sstable
2) sstable with edgecase
3) given your indexratio is 0.1 you need add at least 10 entries into the table to see if index works, I suggest you add 100 entries.